File: eval/fixtures/legacy-directory-backed-asmx-commerce/src/Legacy/Customers/CustomerProfileService.cs
using System.Web.Services;

namespace Acme.Legacy.Customers;

[WebService(Name = "CustomerProfileService", Namespace = "http://legacy.example/customers")]
public class CustomerProfileService : WebService
{
  private readonly CustomerProfileManager _manager = new CustomerProfileManager();

  [WebMethod(MessageName = "ReviewProfileRisk")]
  public string ReviewProfileRisk(string customerId)
  {
    return _manager.ReviewRisk(customerId);
  }
}

File: eval/fixtures/legacy-directory-backed-jaxws-commerce/src/main/java/com/acme/legacy/customers/CustomerProfileService.java
package com.acme.legacy.customers;
import javax.jws.WebMethod;
import javax.jws.WebService;

@WebService(serviceName = "CustomerProfileService")
public class CustomerProfileService {
  private final CustomerProfileManager customerProfileManager = new CustomerProfileManager();

  @WebMethod(operationName = "ReviewProfileRisk")
  public ProfileRiskResult reviewProfileRisk(String customerId) {
    return customerProfileManager.reviewRisk(customerId);
  }
}
